perl/buildlib/Lucy/Build/Binding/Object.pm

```diff
@@ -78,18 +78,22 @@ CODE:
         ints_av = (AV*)SvRV(ints_sv);
     }
     if (ints_av && SvTYPE(ints_av) == SVt_PVAV) {
-        int32_t size  = av_len(ints_av) + 1;
-        int32_t *ints = (int32_t*)CFISH_MALLOCATE(size * sizeof(int32_t));
-        int32_t i;
+        int64_t size = av_len(ints_av) + 1;
+        int32_t *ints;
+        int64_t i;
```

Comment:
I just realized that Perl switched from `I32` to `SSize_t` for array sizes in 5.20. Ideally, we should add a typedef depending on the Perl version and use it for `size` and `i`. But this can be done later.